### PR TITLE
docs: superset and metabase integration

### PR DESCRIPTION
**Pull request by sunng87**
## What's Changed in this PR

Add Apache Superset setup guide.

## Checklist

- [ ] Please confirm that all corresponding versions of the documents have been revised.
- [ ] Please ensure that the content in `sidebars.ts` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

File: docs/greptimecloud/integrations/metabase.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Metabase
+
+[Metabase](https://github.com/metabase/metabase) is an open source BI tool that
+written in Clojure. You can configure GreptimeDB as a metabase data source from
+a community driver plugin.
+
+See [our docs](https://docs.greptime.com/user-guide/integrations/metabase) for
```
Comment (nicecui) [inline docs/greptimecloud/integrations/metabase.md:7]:
```suggestion
See [docs](https://docs.greptime.com/user-guide/integrations/metabase) for
```

File: docs/user-guide/integrations/metabase.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## Installation
+
+Download the driver plugin file `greptimedb.metabase-driver.jar` from its
+[release
+page](https://github.com/greptimeteam/greptimedb-metabase-driver/releases/latest/). Copy
+the jar file to `plugins/` of metabase's working directory. It will be
+discovered by Metabase automatically.
+
+## Add GreptimeDB as Database
```
Comment (nicecui) [inline docs/user-guide/integrations/metabase.md:15]:
```suggestion
## Add GreptimeDB as database
```

File: docs/user-guide/integrations/metabase.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+To add GreptimeDB database, select *Settings* / *Admin Settings* / *Databases*,
+click *Add Database* button and select GreptimeDB from *Database type*.
+
+You will be asked to provide host, port, database name and authentication
+information.
+
+- Use Greptime's Postgres protocol port `4003` as port. If you changed the
+  defaults, use you own settings.
+- Username and password are optional if you didn't enable authentication.
```
Comment (nicecui) [inline docs/user-guide/integrations/metabase.md:25]:
```suggestion
- Username and password are optional if you didn't enable [authentication](/user-guide/operations/authentication.md).
```

File: docs/user-guide/integrations/overview.md [hunk 4]
```diff
@@ -6,5 +6,5 @@ The subsequent sections offer comprehensive guidance on integrating GreptimeDB w
 - [Prometheus](./prometheus.md)
 - [Vector](./vector.md)
 - [Grafana](./grafana.md)
+- [Superset](./superset.md)
```
Comment (nicecui) [inline docs/user-guide/integrations/overview.md:9]:
```suggestion
- [Metabase](./metabase.md)
- [Superset](./superset.md)
```

File: docs/user-guide/integrations/superset.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+greptimedb-sqlalchemy
+```
+
+Start Superset services:
+
+```bash
+docker compose -f docker-compose-non-dev.yml up
+```
+
+### Running Superset Locally
```
Comment (nicecui) [inline docs/user-guide/integrations/superset.md:27]:
```suggestion
### Running Superset locally
```

File: docs/user-guide/integrations/superset.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+If you are [running Superset from
+pypi](https://superset.apache.org/docs/installation/pypi), install our extension
+to the same environment.
+
+```bash
+pip install greptimedb-sqlalchemy
+```
+
+## Adding GreptimeDB Database
```
Comment (nicecui) [inline docs/user-guide/integrations/superset.md:37]:
```suggestion
## Adding GreptimeDB database
```

File: docs/user-guide/integrations/superset.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+Add database and select *GreptimeDB* from list of supported databases.
+
+Follow the SQLAlchemy URI pattern to provide your connection information:
+
+```
+greptimedb://<username>:<password>@<host>:<port>/<database>
+```
+
+- Ignore `<username>:<password>@` if you don't have authentication enabled.
```
Comment (nicecui) [inline docs/user-guide/integrations/superset.md:49]:
```suggestion
- Ignore `<username>:<password>@` if you don't have [authentication](/user-guide/operations/authentication.md) enabled.
```

**Review by nicecui — COMMENTED** [review]
- To simplify the consistency maintenance cost of the document, please add zh documents to `/i18n/zh/docusaurus-plugin-content-docs/current` in this PR if you can.
- Please use two spaces for indentation in typescript files

File: i18n/zh/docusaurus-plugin-content-docs/current/user-guide/integrations/metabase.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
+# Metabase
+
+[Metabase](https://github.com/metabase/metabase) 是一个用 Clojure 编写的开源 BI
+工具，可以通过社区维护的数据库驱动将 GreptimeDB 添加到 Metabase。
+
+## 安装
+
+从 [发布
+页](https://github.com/greptimeteam/greptimedb-metabase-driver/releases/latest/)
+下载最新的驱动插件文件 `greptimedb.metabase-driver.jar`。并将文件拷贝到 Metabase
```
Comment (nicecui) [inline i18n/zh/docusaurus-plugin-content-docs/current/user-guide/integrations/metabase.md:10]:
```suggestion
下载最新的驱动插件文件 `greptimedb.metabase-driver.jar`，并将文件拷贝到 Metabase
```

File: i18n/zh/docusaurus-plugin-content-docs/current/user-guide/integrations/superset.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+
+启动 Supertset 服务：
+
+```bash
+docker compose -f docker-compose-non-dev.yml up
+```
+
+### 本地运行 Superset
+
+加入你通过 [Pypi 包安装和运行
```
Comment (nicecui) [inline i18n/zh/docusaurus-plugin-content-docs/current/user-guide/integrations/superset.md:28]:
```suggestion
假如你通过 [Pypi 包安装和运行
```

**Review by nicecui — APPROVED** [review]
LGTM

**Review by fengjiachun — APPROVED** [review]
LGTM

**Pull request merged**